src/graph.js

```diff
@@ -57,8 +58,8 @@ export default class Graph {
     coords.length = requiredNumOfPoints;
 
     this.coords = this._calcPoints(coords);
-    this.min = Math.min(...this.coords.map(item => Number(item[V])));
-    this.max = Math.max(...this.coords.map(item => Number(item[V])));
+    this.min = Math.min(...this._calcPoints(coords).map(item => Number(item[V])));
```

Comment:
Can you explain why do you need to calculate points again?

Reply:
These min and max values are used to compute the scale of the bar graph. If you use the initial data but not the values transformed thru the "aggregate_func" there is a scale issue. It's not too big of a deal with the min/max/avg/first/last but  with the delta function the scale is way to small.

Reply:
Maybe I'm missing something... byt sorry, I still don't understand why it is required. In line 60 you call `this._calcPoints(coords)` so why do you need to repeat it in both lines below if the input collection doesn't change. 

Call to this function with the same input collection should produce exactly the same result every time.

Reply:
You're right ! I updated the code in commit e7f5100